### ISSUE TITLE
Adding Google Analytics Script

### ISSUE THREAD

**Issue by hanabadler** [TheHive-Project/TheHive#410]
Hi thehive group

we are using your app and its working great.we are using the REST api to push data into, and we are using the webhooks for collecting audit trail.

we want to use google analytics in the app.
this requires putting the GA script before the </head> section 

Can you advise how to do it? 
thanks
Hana

**Comment by nadouani** [comment]
Hello @hanabadler,

I don't think we will vote for including a google analytics snippet on TheHive by default, but you can fork it and add the GA missing part to `https://github.com/TheHive-Project/TheHive/blob/master/ui/app/index.html`file.

You will need to build TheHive from sources.

What's the use case of using GA on an app like TheHive?

Nabil

**Comment by hanabadler** [comment]
Hi

the GA is only an example . we have internal analytic engine , and the JavaScript is added for every web application on our environment.
we didn't built from source, and we updated the index.html inside the jar.
of course, once an update is done, it will override the changes in the index.html and we will need to modify over and over...

it will be nice to have an option to place it on the conf files (no need for any GUI).
thanks
Hana

**Comment by saadkadhi** [comment]
@hanabadler I've added your request to #149. We will track it there. Scheduled for implementation in Cerana 2 (3.2).
